packages/api/src/memberContext.ts

```diff
@@ -242,7 +242,12 @@ export class MemberContext implements MemberContext {
           (paidUntil !== null && periods[periods.length - 1].endsAt > paidUntil))
       ) {
         const period = periods[periods.length - 1]
-        return await this.loaders.invoicesByID.load(period.id)
+        const invoice = await this.dbAdapter.invoice.getInvoiceByID(period.invoiceID)
```

Comment:
What's the reason we avoid the cache now?

Reply:
no particular reason. But i'm pretty sure in this context we don't gain any speed if we use the cache.